cli/src/semdep/parse_lockfile.py

```diff
@@ -661,14 +661,43 @@ def parse_manifest(text: str) -> Set[str]:
         )
 
 
+def parse_pom_tree(tree_str: str, _: Optional[str]) -> Iterator[FoundDependency]:
+    def package_index(line: str) -> int:
+        i = 0
+        while line[i] in ["+", "-", " ", "\\"]:
+            i += 1
+        return i
+
+    def depth(package_index: int) -> int:
+        return package_index // 3 + 1
```

Comment:
Are you sure this is consistent?

Reply:
I should make the example file bigger but yes I am